File: planttech/src/main/java/com/planttech/domain/SensorControlTf.java
package com.planttech.domain;

import java.sql.Timestamp;
import lombok.Data;

@Data
public class SensorControlTf {

	private int sensorControlNo;
	private int userNo;
	
	private int waterPumpTf;
	private int ledTf;

	private Timestamp sensorUpdateTimestamp;

}

File: Plant/app/src/main/java/com/example/plant/model/SensorControlTf.kt
package com.example.plant.model

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime
import java.util.Date

data class SensorControlTf(
    //@SerializedName("sensorControlNo") val sensorControlNo:Int,
    @SerializedName("userNo") val userNo:Int,
   // @SerializedName("waterPumpTf") val waterPumpTf:Int,
    @SerializedName("ledTf") val ledTf:Int
    //@SerializedName("sensorUpdateTimestamp") val sensorUpdateTimestamp: LocalDateTime,

){
    override fun toString(): String {
        return "userNo : $userNo\n" +
                //"waterPumpTf : $waterPumpTf\n" +
                "ledTf : $ledTf\n"
                //"sensorUpdateTimestamp : $sensorUpdateTimestamp\n"

    }
}